roles/variables/vars/main.yml

```diff
@@ -1277,16 +1304,31 @@ ksql_properties:
                             'confluent.monitoring.interceptor.', ksql_truststore_path, ksql_truststore_storepass, public_certificates_enabled, ksql_keystore_path, ksql_keystore_storepass, ksql_keystore_keypass,
                             false, sasl_plain_users_final.ksql.principal, sasl_plain_users_final.ksql.password, sasl_scram_users_final.ksql.principal, sasl_scram_users_final.ksql.password, sasl_scram256_users_final.ksql.principal, sasl_scram256_users_final.ksql.password,
                             kerberos_kafka_broker_primary, ksql_keytab_path, ksql_kerberos_principal|default('ksql'),
-                            false, ksql_ldap_user, ksql_ldap_password, mds_bootstrap_server_urls, false,'', '', '', '') }}"
+                            false, ksql_ldap_user, ksql_ldap_password, mds_bootstrap_server_urls, oauth_enabled and (not ldap_with_oauth_enabled),ksql_oauth_user, ksql_oauth_password, oauth_groups_scope, oauth_token_uri) }}"
   rbac:
     enabled: "{{rbac_enabled}}"
     properties:
-      ksql.security.extension.class: io.confluent.ksql.security.KsqlConfluentSecurityExtension
-      ksql.authentication.plugin.class: io.confluent.ksql.security.VertxBearerOrBasicAuthenticationPlugin
       public.key.path: "{{rbac_enabled_public_pem_path}}"
       confluent.metadata.bootstrap.server.urls: "{{mds_bootstrap_server_urls}}"
+      confluent.metadata.http.auth.credentials.provider: OAUTHBEARER
```

Comment:
will this config not clash with rbac_ldap as this rbac and rbac_ldap section both have different value for confluent.metadata.http.auth.credentials.provider OAUTHBEARER & BASIC

Reply:
thanks for pointing this out, will fix this